ethereum_history_api/oracles/README.md

```diff
@@ -1,3 +1,31 @@
 ## Noir Ethereum history api oracles
 
 TypeScript oracle server that provides data for circuits.
+
+### Preparing fixtures
+
+Fixtures are data used for testing. They are downloaded and kept as local data. There are two types of fixtures: noir and js. They are pepared in different format and are used in different tests.
+
+To generate data for noir tests run:
+
+```sh
+yarn prepare-noir-fixtures
+```
+
+This command will delete current `circuits/lib/src/fixtures` catalog and it will create a new one with new data prepared inside it. Data will be prepared in noir format.
```

Comment:
Maybe delete this paragraph and insert info about regenerating the folder in the paragraph after prepare-js-fixtures?

Reply:
Or maybe let's not mention that data is being deleted and generated from scratch at all?